drizzle/migration/schema.ts

```diff
@@ -1,26 +1,10 @@
-import { pgEnum } from "drizzle-orm/pg-core";
+import { pgEnum } from "drizzle-orm/pg-core"
```

Comment:
We don't have pre-commit hooks in this repo. So, we'll need to make sure we manually run 
`npm run format`

(The lack of semi-colon makes me think we haven't run the formatter)